components-core/src/main/java/com/airwallex/android/core/Airwallex.kt

```diff
@@ -310,6 +360,30 @@ class Airwallex internal constructor(
         )
     }
 
+    /**
+     * Retrieve available payment consents
+     *
+     * @param params [RetrieveAvailablePaymentConsentsParams] used to retrieve all [PaymentConsent]
+     * @param callback [AirwallexCallback] A callback interface to handle the success or failure of the network request.
+     */
+    fun retrieveAvailablePaymentConsents(
+        params: RetrieveAvailablePaymentConsentsParams,
+        callback: AirwallexCallback<Page<PaymentConsent>>
+    ) {
+        activity.lifecycleScope.launch(Dispatchers.IO) {
```

Comment:
might not need to do this since AirwallexApiRepository does.

Reply:
ok